CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Supercontrast
+
+We welcome contributions to the Supercontrast project! To get started, please follow these steps:
+
+### 1. Fork the Repository
+
+First, fork the repository on GitHub to create your own copy. Then, clone your fork to your local machine:
+
+```bash
+git clone https://github.com/YOUR_USERNAME/supercontrast.git
+```
+
+### 2. Install the Package
+Navigate to the project directory and install the package in development mode:
+
+``` 
+cd supercontrast
+pip install -e .[dev]
+
+```
+
+### 3. Run Linting

```

Comment:
Small nit here, but we should run the linting after our contributors have made changes to the codebase not before :)